pydoctor/test/test_astbuilder.py

```diff
@@ -1988,3 +1988,16 @@ def test_constant_override_do_not_warns_when_defined_in_module_docstring(systemc
     assert ast.literal_eval(attr.value) == 99
     captured = capsys.readouterr().out
     assert not captured
+
+@systemcls_param
+def test__name__equals__main__is_skipped(systemcls: Type[model.System]) -> None:
+    mod = fromText('''
```

Comment:
I think that it would be nice to have a docstring decribing the purpose and scope of this test.
When you end up with hundred of tests is hard to tell which tests is for which use case/functionality.

```suggestion
def test__name__equals__main__is_skipped(systemcls: Type[model.System]) -> None:
    """
    The code included inside the C{__main__} block is excluded from the API docs.
    Any outside code is part of the API.
    """
    mod = fromText('''
```